features/step_definitions/common.ts

```diff
@@ -162,6 +162,17 @@ Then(
   },
 );
 
+Then(
+  "The output message should match with the data below:",
+  async function (table) {
+    const records = table.raw();
+    records.forEach((record: string[]) => {
+      const values = record.map((field: string) => `"${field}"`).join(",");
```

Comment:
@tuanphamcybozu 

If importing is an empty string for the field, the validation will fail even It's expected behavior.

Please check the data below:
```
| Text   | Number |
| Alice  |        |
| Bob    | 20     |
| Jenny  | 30     |
```